packages/polaris-viz/src/components/SimpleBarChart/Chart.tsx

```diff
@@ -110,64 +114,71 @@ export function Chart({
-              style={{
-                opacity,
-                transform,
-              }}
-            >
-              <HorizontalGroup
-                areAllNegative={areAllNegative}
-                ariaLabel={ariaLabel}
-                barHeight={barHeight}
-                containerWidth={width}
-                data={data}
-                groupHeight={groupHeight}
-                id={id}
-                index={index}
-                isSimple
-                isStacked={isStacked}
-                name={name}
-                stackedValues={stackedValues}
-                xAxisOptions={xAxisOptions}
-                xScale={xScale}
-                yAxisOptions={yAxisOptions}
-                zeroPosition={zeroPosition}
-              />
-            </animated.g>
-          );
-        })}
-      </ChartElements.Svg>
+  return (
+    <div className={styles.SimpleBarChartWrapper} style={containerStyle}>
+      <div className={styles.BarChart}>
+        <ChartElements.Svg height={height} width={width}>
+          <GradientDefs
+            direction="horizontal"
+            gradientUnits={isStacked ? 'objectBoundingBox' : 'userSpaceOnUse'}
+            id={id}
+            seriesColors={seriesColors}
+            size={isStacked ? '100%' : `${width}px`}
+          />
+
+          {transitions((style, item, _transition, index) => {
+            const {opacity, transform} = style as HorizontalTransitionStyle;
+            const name = item.key ?? '';
+            const ariaLabel = getAriaLabel({
+              seriesIndex: item.index,
+              key: data[0].data[item.index]?.key,
+            });
+
+            return (
+              <animated.g
+                key={`group-${name}`}
+                style={{
+                  opacity,
+                  transform,
+                }}
+              >
+                <HorizontalGroup
+                  areAllNegative={areAllNegative}
+                  ariaLabel={ariaLabel}
+                  barHeight={barHeight}
+                  containerWidth={width}
+                  data={data}
+                  groupHeight={groupHeight}
+                  id={id}
+                  index={index}
+                  isSimple
+                  isStacked={isStacked}
+                  name={name}
+                  stackedValues={stackedValues}
+                  xAxisOptions={xAxisOptions}
+                  xScale={xScale}
+                  yAxisOptions={yAxisOptions}
+                  zeroPosition={zeroPosition}
+                />
+              </animated.g>
+            );
+          })}
+        </ChartElements.Svg>
+      </div>
 
       {showLegend && (
         <LegendContainer
           colorVisionType={COLOR_VISION_SINGLE_ITEM}
           data={legend}
           onDimensionChange={setLegendDimensions}
           renderLegendContent={renderLegendContent}
+          position={legendPosition}
         />
       )}
-    </ChartElements.Div>
```

Comment:
@envex I used a plain `<div>` because `<ChartElements.Div>` currently only accepts width and height params, but I needed to set some flex styles to achieve the alignment. Would we be better off adding new prop/props to `<ChartElements.Div>` to configure this (e.g. `flexDirection` and `alignItems` or similar)?

Reply:
Added a `style` prop in 14a1cdae5221d4119d121e941c988750a31be87f